lib/src/broker/tokens.dart

```diff
@@ -247,6 +247,17 @@ class TokenNode extends BrokerNode {
     }
     links = null;
   }
+
+  void updateLinks() {
+    if (links == null) return;
+    for (Object path in links) {
+      if (path is! String) continue;
+      Object nd = provider.getNode(path as String);
+      if (nd is! RemoteLinkRootNode) continue;
+      (nd as RemoteLinkRootNode).configs[r'$$group'] = group;
+      (nd as RemoteLinkRootNode).updateList(r'$$group');
+    }
+  }
 }
```

Comment:
need to check if any if the link is changed, and after the loop save conns.json if needed